README.md

```diff
@@ -1,22 +1,42 @@
-## Istio Installer
-This image is used by the rancher istio dev-chart to intall istio with various configurations 
+# Istio Installer
+This image is used by the [rancher-istio chart](https://github.com/rancher/charts "rancher-istio chart") to intall istio with various configurations.
 
-Scripts include install, upgrade and uninstall. 
+This installer uses shell scripts to run istioctl install, upgrade and uninstall commands. 
 
 ## Build
-Image is built when a new tag is created and pushed to repository.
+The istio-installer image is built when a new tag is created and pushed to repository.
 
+####  How to tag
+The istio installer uses the following version format for tags: `<x.x.x>-rancher<x>`
+
+The `<x.x.x>` should be set the version of istio the installer is going to be used for. This can be found in the Dockerfile as the value for the `ENV ISTIO_VERSION`
```

Comment:
```suggestion
The `<x.x.x>` should be set to the version of istio the installer is going to be used for. This can be found in the Dockerfile as the value for the `ENV ISTIO_VERSION`
```

Reply:
fixed in most recent commit

Reply:
added in the most recent commit